step-capstone/src/components/ItemForm.js

```diff
@@ -111,22 +111,22 @@ export default class ItemForm extends React.Component {
                         <Tab label="Hotel" />
                     </Tabs>
                 </Paper>
-
             )
         }
         return null;
     }
 
     render() {
         return (
-            <Card id="add-form-container">
+            <Card id="add-form-container" style={{height: "350px"}}>
                 <CardContent>
                     {this.renderTab()}
                     {this.getForm()}
                 </CardContent>
                 <CardActions>
                     <Button onClick={this.state.onClose} size="small">Cancel</Button>
                     <Button onClick={this.handleSave} size="small">Save</Button>
+                    <Button onClick={() => this.props.onRemoveItem(this.state.data.id)} size="small">Delete</Button>
```

Comment:
why do we need to be able to delete the item form?

Reply:
oh i see. this is the same form that appears when you click on the event. not sure how difficult it would be to change, but now, when you click the plus button, it also has a delete button, which doesnt do anything.

Reply:
The functionality of this Delete button is actually to delete the Travel Object.

Reply:
I agree with @zirving in that we should think of the case when we're adding an object, in which we would have no reason to present the user with a delete button. Maybe only render this button when we're editing?

Reply:
Ah, turns out that's what Zach means. Okay, I will make changes so that the button only renders when editing.

Reply:
Done